config_pgcluster.yml

```diff
@@ -69,6 +69,8 @@
     - name: Make sure the gnupg and apt-transport-https packages are present
       apt:
         pkg:
+          - curl
```

Comment:
> Make sure the gnupg and apt-transport-https packages are present

the curl package is already defined in vars/debian - https://github.com/vitabaks/postgresql_cluster/blob/master/vars/Debian.yml#L43